octoprint_findmymrbeam/__init__.py

```diff
@@ -125,6 +128,45 @@ def _get_dns_domain_names():
 			dns_domain_names = list(set(dns_domain_names))
 			return dns_domain_names
 
+	@staticmethod
+	def _get_netifaces():
+		"""
+		Get the network interfaces for the device including the IPv4 and IPv6 and return them back
+
+		Returns:
+			list : network interfaces list
+
+		"""
+		netifaces_list = []
+
+		# 1. Get all the interfaces names that comply to the regex mentioned, only wlan and eth interfaces
+		iface_names_list = \
+			[iface_name for iface_name in netifaces.interfaces() if re.match(r"eth\d.*|wlan\d.*", iface_name) is not None]
```

Comment:
```suggestion
			[iface_name for iface_name in netifaces.interfaces() if re.match(r"eth\d.*|wlan\d.*", iface_name)]
```

Reply:
![image](https://user-images.githubusercontent.com/85931758/200260731-689c0950-c599-48f5-bdf6-58e1e38f062c.png)

Reply:
I was thinking of this: https://mr-beam.atlassian.net/wiki/spaces/SOFTWARE/pages/229310465/Python+-+Language+Rules#14.-True%2FFalse-evaluations
But as the second point states, we should keep this as it was. I will resolve